extmod/machine_i2c.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
```

Comment:
This file has to be removed.

Reply:
Do we need to add anything more while removing this? Just removing this file makes the build fail.

Reply:
Done